ptp/components/models/vqa/attention.py

```diff
@@ -137,8 +144,12 @@ def forward(self, data_dict):
         attention_enc_img = apply_attention(enc_img, attention) # [48, 2048, 7, 7], [48, 2, 7, 7]
         # print("attention im", attention_enc_img.shape)
 
-        # Fusion -- Concatenate attention-weighted image encodings and question encodings.
-        outputs = torch.cat([attention_enc_img, latent_q], dim=1)
+        if(self.output_mode == 'Image'):
+        # Output attention-weighted image encodings
+            outputs = attention_enc_img
+        elif(self.output_mode == 'None'):
```

Comment:
I think 'None' is misleading - seems that the component will return something, right...?

Reply:
Yes, will change this :) thanks!

Reply:
Hey! I fixed this 'output_mode' to take two states: (1) 'Image' --- to return only attention-weighted image, and (2) 'Fusion' -- to return a concatenated output of attention-weighted image and question.

Hope this can be merged now :)